test/core/end2end/h2_ssl_cert_test.cc

```diff
@@ -304,11 +288,7 @@ static void shutdown_client(grpc_end2end_test_fixture* f) {
 static void end_test(grpc_end2end_test_fixture* f) {
   shutdown_client(f);
   shutdown_server(f);
-
-  grpc_completion_queue_shutdown(f->cq);
-  drain_cq(f->cq);
   grpc_completion_queue_destroy(f->cq);
-  grpc_completion_queue_destroy(f->shutdown_cq);
```

Comment:
Unfortunately `grpc_completion_queue_destroy(f->shutdown_cq)` must still be called though - else its a memory leak
(I know its a very weird contract - but the fixture expects all tests to call destroy on `f->shutdown_cq` during test-teardown)

Reply:
so, even if it's not being used, it must be created and destroyed?

Reply:
This is a standalone test btw. It's not mixed with other filters/tests. If I remove the create, then I don't need to destroy either.

Reply:
yeah..if you removed the create you wouldn't need to destroy. I missed the "removing the create" part :)